metadata-service/factories/src/main/java/com/linkedin/gms/factory/auth/AuthorizerChainFactory.java

```diff
@@ -52,12 +53,17 @@ public class AuthorizerChainFactory {
   protected AuthorizerChain getInstance() {
     // Init authorizer context
     final AuthorizerContext ctx = initAuthorizerContext();
+
     // Extract + initialize customer authorizers from application configs.
     final List<Authorizer> authorizers = new ArrayList<>(initCustomAuthorizers(ctx));
-    // Add the DataHub core policies-based Authorizer - this one should always be enabled.
-    this.dataHubAuthorizer.init(Collections.emptyMap(), ctx);
-    authorizers.add(this.dataHubAuthorizer);
-    return new AuthorizerChain(authorizers);
+
+    if (configurationProvider.getAuthorization().getDefaultAuthorizer().isEnabled()) {
```

Comment:
Why are we supporting this?

Reply:
As discussed over a call. It is to allow customers to control the authorization